pkg/util/error.go

```diff
@@ -63,3 +63,17 @@ func IsNotFound(err error) bool {
 	}
 	return false
 }
+
+func IsConnectionRefusedOrUnauthorized(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	if strings.Contains(strings.ToLower(err.Error()),
+		"connection refused") || strings.Contains(strings.ToLower(err.Error()),
+		"unauthorized") {
```

Comment:
```suggestion
         msg := strings.ToLower(err.Error())
         if strings.Contains(msg, "connection refused") ||
                 strings.Contains(msg, "unauthorized") {
                 return true
```

Reply:
Actually, you can improve it even further: Just return the boolean directly, without the `if` statement:
```
         msg := strings.ToLower(err.Error())
         return strings.Contains(msg, "connection refused") ||
                 strings.Contains(msg, "unauthorized")

```